mix.exs

```diff
@@ -10,23 +10,14 @@ defmodule Thrifter.Mixfile do
      deps: deps]
   end
 
-  # Configuration for the OTP application
-  #
-  # Type "mix help compile.app" for more information
   def application do
     [applications: [:logger]]
   end
 
-  # Dependencies can be Hex packages:
-  #
-  #   {:mydep, "~> 0.3.0"}
-  #
-  # Or git/path repositories:
-  #
-  #   {:mydep, git: "https://github.com/elixir-lang/mydep.git", tag: "0.1.0"}
-  #
-  # Type "mix help deps" for more examples and options
   defp deps do
-    []
+    [
+      {:riffed, git: "git@github.com:renderedtext/riffed",
+        ref: "origin/master"},
```

Comment:
Lets use pinterest's riffed. We don't want to maintain a fork of riffed if it is not mandatory.